Refresh

jakirkham opened the pull request:
* Re-rendered with conda-smithy 2.4.5.
* Update license metadata to include version.
* Package the license file.
* Switch to `sha256`.
* Use `skip` to avoid unsupported Python versions.
* Use `script` for install instead of build scripts.
* Bump build number to `2`.

jakirkham committed:
MNT: Re-rendered with conda-smithy 2.4.5

jakirkham committed:
Update license metadata

Drop redundant use of "License" in `license` name. Also add the license
version.

jakirkham committed:
MNT: Re-rendered with conda-smithy 2.4.5

Now that the license metadata is updated, re-render to propagate this
change to the README.

jakirkham committed:
Package the license file

Make sure the license file is included in the conda package for
inspection by end users.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

jakirkham committed:
Template out name, version, build number, and hash

To simplify filling out the package information (e.g. download URL), set
the name, version, build number, and hash with Jinja at the top and
template the package metadata using these values.

jakirkham committed:
Use sha256 instead of md5

The `md5` checksum is broken. As PyPI supplies the `sha256` checksum,
which is much more robust, switch to using `sha256` instead.

jakirkham committed:
Use `skip` to determine Python versions built

Instead of pinning the Python version in `build` and `run`, use `skip`
to set which Python versions should or should not be built and drop the
version spec.

jakirkham committed:
Use `script` for the install step

As `script` works reliably on all platforms for some time now and is the
preferred conda-forge mechanisms for simple installs, use `script` and
drop the `build.sh` and `bld.bat` scripts.

jakirkham committed:
Bump build number to 2

Rebuild now that the license file will be packaged and there have been
some light metadata adjustments.
